src/ripple/protocol/SystemParameters.h

```diff
@@ -59,6 +60,18 @@ systemCurrencyCode()
 /** The XRP ledger network's earliest allowed sequence */
 static std::uint32_t constexpr XRP_LEDGER_EARLIEST_SEQ{32570};
 
+/** The minimum amount of support an amendment should have.
+
+    @note This value is used by legacy code and will become obsolete
+          once the fixAmendmentMajorityCalc amendment activates.
+*/
+constexpr std::ratio<204, 256> amendmentSuperMajorityThresholdPre3396;
+
+constexpr std::ratio<80, 100> amendmentSuperMajorityThresholdPost3396;
```

Comment:
Now that the `fix` has been renamed these names don't make as much sense as they used to.  There aren't any other mentions of 3396 in the source code.  I don't have any great suggestions, but maybe...

- `preFixAmendmentMajorityCalcThreshold` and `postFixAmendmentMajorityCalcThreshold` or
- `preFixAmendmentMajorityCalcSuperMajority` and `postFixAmendmentMajorityCalcSuperMajority`

would be a little easier to figure out.  You may have better ideas than these.

Reply:
Done